SU-HSDO/suhumsci
# ARCH-174 reduced font size for category field in horizontal cards

# READY FOR REVIEW

# Summary
- Reduce the font size of category field in horizontal cards

# Needed By (Date)
- End of sprint (7/19)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that category field in horizontal cards is using the 1.7rem font size (17px)

# Affected Projects or Products
- Archeaolohy
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-174
- BUGHERD-28

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
